cache_images/fedora-netavark_packaging.sh

```diff
@@ -61,7 +61,6 @@ INSTALL_PACKAGES=(\
     zip
 )
 
-# TODO: Remove this when all CI should test with Netavark/Aardvark by default
 EXARG="--exclude=netavark --exclude=aardvark-dns --exclude=cargo --exclude=rust"
```

Comment:
This may have been on purpose (only removing the comment).  This specific script (`fedora-netavark_packaging.sh`) is only used for building the CI VM images used by netavark & aardvark-dns, nowhere else.  For those use-cases, CI is using runtime-produced & installed netavark/aardvark-dns binaries.  So we definitely don't want the RPMs installed.  I'll double check the other `TODO: Remove this` in the diff (here) and make sure they're intentional...